src/custom-types.json

```diff
@@ -236,6 +236,14 @@
   "application/x-7z-compressed": {
     "compressible": false
   },
+  "application/x-arj": {
+    "compressible": false,
+    "extensions": ["arj"],
+    "sources": [
+      "https://en.wikipedia.org/wiki/List_of_archive_formats",
+      "http://www.wussu.com/various/mimetype.htm"
```

Comment:
Apologies on the Wikipedia link. I missed that note.

I did some digging last night, but have not come up (yet) with an official source. An attempt to email the author (Robert Jung, http://www.arjsoftware.com/) resulted in an email bounce.

A few notes that may be of interest that lead me to `application/x-arj` vs some of the others that have floated around over the years:
* [file(1) uses it](https://opensource.apple.com/source/file/file-37/file/magic/Magdir/archive)
* Most Linux desktop relations seem to use this (for icons, launchers, etc. integration)
* Various references to mime types in older documentation (e.g. the second link in sources I provided is from 1998)

Some issues with the above: 
`application/arj` is also referenced here and there commonly. For example, https://goo.gl/Wl1XoQ (1999). 

I did find a very old email reference for Robert in an original ARJ distribution from the 90's. I have mailed him & hope to hear back (hasn't bounced back yet!)

Thanks!